taxonomy/models.py

```diff
@@ -16,25 +19,46 @@ class Vocabulary(models.Model):
     FREE_TAGGING = "f"
 
     repository = models.ForeignKey(Repository, on_delete=models.PROTECT)
-    name = models.TextField()
-    description = models.TextField()
+    name = models.CharField(max_length=256)
+    slug = models.CharField(max_length=256, unique=True)
+    description = models.TextField(
+        help_text=_("Describe how content authors should use this vocabulary")
+    )
     required = models.BooleanField()
-    vocabulary_type = models.CharField(max_length=1, choices=(
-        (MANAGED, "managed"),
-        (FREE_TAGGING, "free tagging")
-    ))
+    vocabulary_type = models.CharField(
+        max_length=1,
+        choices=(
+            (MANAGED, _("Managed")),
+            (FREE_TAGGING, _("Tag Style (on the fly)"))
+        ),
+        default="m",
+        help_text=_("Should terms be created in advance or on the fly?")
```

Comment:
consider changing this message to, "Select whether vocabulary terms are curated or created by users on-the-fly."  I am trying to find a help message that isn't a question.